src/open_inwoner/accounts/backends.py

```diff
@@ -158,6 +162,31 @@ def _check_candidate_backend(self) -> bool:
             OpenIDEHerkenningConfig,
         )
 
+    def _store_vestigingsnummer_in_session(
+        self, claims: JSONObject, user_created: bool = False
+    ):
+        """Get company vestigingsnummer from OIDC claims & store in session"""
+
+        eherkenning_config = self.config_class.get_solo()
+
+        if not (branch_number_claims := eherkenning_config.branch_number_claim):
+            return
```

Comment:
In fact, the `_check_candidate_backend` already takes care of this, and since the config class provides a default for the claim field, this is guaranteed to exist. Will simplify